frontend/src/App.css

```diff
@@ -36,3 +36,13 @@
     transform: rotate(360deg);
   }
 }
+
+img {
```

Comment:
What do you think about creating a Tools.css and putting this inside of there. I think that if its placed here, it will affect all img elements and all classes that use the jumbotron class. You might have to change the classname from jumbotron to something else so that not all jumbotrons are affected.

Reply:
I tried that earlier but didn't realize I had to import the css file into my Tools.js file so it should work now. Also when you say change the class name why does it affect the other components if its only imported into the Tools component.